test/e2e/e2e.go

```diff
@@ -2444,3 +2444,77 @@ var _ = ginkgo.Describe("e2e delete databases", func() {
 		//table.Entry("when delete south db on ovnkube-db-2", 2, []string{southDBFileName}),
 	)
 })
+
+var _ = ginkgo.Describe("e2e IGMP validation", func() {
+	const (
+		svcname              string = "igmp-test"
+		ovnNs                string = "ovn-kubernetes"
+		port                 string = "8080"
+		ovnWorkerNode        string = "ovn-worker"
+		ovnWorkerNode2       string = "ovn-worker2"
+		mcastGroup           string = "224.1.1.1"
+		multicastListenerPod string = "multicast-listener-test-pod"
+		multicastSourcePod   string = "multicast-source-test-pod"
+		tcpdumpFileName      string = "tcpdump.txt"
+		retryTimeout                = 5 * time.Minute // polling timeout
+	)
+	var (
+		tcpDumpCommand = []string{"bash", "-c",
+			fmt.Sprintf("apk update; apk add tcpdump ; tcpdump multicast > %s", tcpdumpFileName)}
+		// Multicast group (-c 224.1.1.1), UDP (-u), TTL (-T 2), during (-t 3000) seconds, report every (-i 5) seconds
+		multicastSourceCommand = []string{"bash", "-c",
+			fmt.Sprintf("iperf -c %s -u -T 2 -t 3000 -i 5", mcastGroup)}
+	)
+	f := framework.NewDefaultFramework(svcname)
+	ginkgo.It("can retrieve multicast IGMP query", func() {
+
```

Comment:
nit: can you add a `By` section here saying something like `annotating namespace: %s to enable multicast`

Reply:
Done,thanks.